### PR TITLE
Files list: Add "New Folder.." popup when nothing under mouse cursor

### PR DESCRIPTION
**Pull request by Krakean**
Small issue: currently, if there is nothing under mouse cursor in project's file list panel, when you try to right mouse button click - nothing happens.

After this patch, when you RMB on empty space in files list panel, you will see a popup with "New Folder.." item.
Screenshot: https://i.imgur.com/xvvNpbh.png

### CODE REVIEW

File: editor/filesystem_dock.cpp [hunk 1]
```diff
@@ -1447,6 +1447,15 @@ void FileSystemDock::_files_list_rmb_select(int p_item, const Vector2 &p_pos) {
 	file_options->popup();
 }
 
+void FileSystemDock::_rmb_pressed(const Vector2 &p_pos) {
+	folder_options->clear();
+	folder_options->set_size(Size2(1, 1));
+
+	folder_options->add_item(TTR("New Folder.."), FOLDER_NEW_FOLDER);
```
Comment (akien-mga) [inline editor/filesystem_dock.cpp]: Shouldn't it be `FILE_NEW_FOLDER`?

**Comment by akien-mga** [comment]
Please use `git rebase origin master` to get rid of the unnecessary merge commit in the PR ([PR workflow](docs.godotengine.org/en/latest/community/contributing/pr_workflow.html)).

File: editor/filesystem_dock.cpp (hunk 1 above)
Comment (Krakean) [reply editor/filesystem_dock.cpp]: Well, thought that way too, but works fine even with FOLDER_NEW_FOLDER.

Comment (akien-mga) [reply editor/filesystem_dock.cpp]: Oh nevermind, both options exist indeed.

**Comment by akien-mga** [comment]
Could you rebase to squash the commits into one? See above link for the detailed procedure.

**Comment by Krakean** [comment]
@akien-mga sorry, trying to. will let you know here when I succeed :)

**Comment by Krakean** [comment]
@akien-mga had to create new PR - https://github.com/godotengine/godot/pull/13105, hopefully now its fine.

**Pull request closed without merging**